torch/csrc/lazy/core/hash.cpp

```diff
@@ -89,5 +89,23 @@ std::string HashToString(const hash_t& a) {
   return ss.str();
 }
 
+hash_t Hash(const std::vector<bool>& values){
+  // We can't assume a DataHash size/dataptr approach here bc
+  // vector<bool> can be optimized as vector<bit> and storage details
+  // are decoupled from actual size of 'bool' type
+  hash_t h = 0xad2ed1983bbf2e28;
```

Comment:
@ezyang Wdyt about this? I had a failure in macos build, 

Compiling LazyIr.h is the first time we've exercised Hash(vector<bool) 

I also added a new test (see test_misc.cpp in this diff)

Reply:
uhhh, sure! :) I'm not even sure what you find objectionable about this haha